lib/index.js

```diff
@@ -22,11 +32,33 @@ class Client {
     this.fetch = (...args) => this.queue(fetch.bind(this, ...args))
   }
 
+  setModelsPath (path) {
+    this.modelsPath = path
+  }
+
+  setEntriesPath (path) {
+    this.entriesPath = path
+  }
+
   migrateContentTypes () {
     const models = fs.readdirSync(this.modelsPath).map((f) =>
       require(`${this.modelsPath}/${f}`)
     )
-    return Promise.all(models.map((m) => this.updateContentModel(m)))
+    return Promise.all(models.map((m) => this.updateContentType(m)))
+  }
+
+  migrateEntries () {
+    const entries = fs.readdirSync(this.entriesPath).map((f) =>
+      require(`${this.entriesPath}/${f}`)
+    )
+    return Promise.all(flatten(entries).map((e) => this.updateEntry(e)))
+  }
+
+  seedEntries () {
+    const entries = fs.readdirSync(this.entriesPath).map((f) =>
+      require(`${this.entriesPath}/${f}`)
+    )
+    return Promise.all(flatten(entries).map((e) => this.createEntry(e)))
```

Comment:
Oh I see. I handled the `readdirSync` a little differently. Incorporating both then, what do you think of:
```js
  getEntryStructures () {
    return flatten(this.readDataFromPath(this.entriesPath))
  }

  readDataFromPath (path) {
    return fs.readdirSync(path).map((file) => require(`${path}/${file}`))
  }

  migrateContentTypes () {
    const models = this.readDataFromPath(this.modelsPath)
    return Promise.all(models.map((m) => this.updateContentType(m)))
  }

  migrateEntries () {
    return Promise.all(getEntryStructures().map((e) => this.updateEntry(e)))
  }

  seedEntries () {
    return Promise.all(getEntryStructures().map((e) => this.createEntry(e)))
  }
```